service/r2/routes_test.go

```diff
@@ -204,64 +204,64 @@ func newMockStore() Store {
 	return mockStore{}
 }
 
-func (s mockStore) FetchNamespaces() (*rules.NamespacesView, error) {
-	return &rules.NamespacesView{}, nil
+func (s mockStore) FetchNamespaces() (*models.NamespacesView, error) {
```

Comment:
Can you please create an issue so we can tackle using mockgen to generate mocks for m3ctl to replace the hand-written mocks?

Reply:
https://github.com/m3db/m3ctl/issues/62

👍

Reply:
Sweet thanks.